source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +366,138 @@ provision the following networking API entities, represented in yaml:
+only the dhcpv6-stateless configurations require it.
+
+### Configuration mechanism used for each option
+
+|        Option Name        | dhcpv6-stateful | dhcpv6-stateless |
+|:-------------------------:|:---------------:|:----------------:|
+| DNS recursive name server |       DHCP      |       DHCP       |
+|     Domain Search List    |       DHCP      |       DHCP       |
+|       Network Prefix      |       None      |        RA        |
+|         IP address        |       DHCP      |  Auto generated  |
+|   Network Prefix Length   |        RA       |        RA        |
+|            MTU            |        RA       |        RA        |
+|      Default Gateway      |        RA       |        RA        |
+
+### Supported use cases per dhcpv6 configuration
+
+| Tables                  |           dhcpv6-stateful           | dhcpv6-stateless |
+|-------------------------|:-----------------------------------:|:----------------:|
+| fixed IPs               |              supported              |   not supported  |
+| allowed prefix lengths  |            64 <= X <= 128           |      64 only     |
+| ipv6 privacy extensions |            not supported            |   not supported  |
+| isolated subnet         | if guest routes manually configured |   not supported  |
+
+A brief explanation of each of the use cases described in the tables can be
+found below.
+
+#### Fixed IPs
+
+Fixed IPs are not supported on dhcpv6-stateless since the client itself
+auto-generates the IP address based on the advertised network prefix.
+This means the IP address ***is not*** sent in the DHCPv6 Reply message, thus
+making the guest VM IP address configuration impossible.
+
+#### Allowed prefix lengths
+
+The [slaac](https://tools.ietf.org/html/rfc4862) RFC indicates that all network
+prefixes **are** 64 bit long. Since on DHCPv6 stateless the IP prefix
+configuration is done as in slaac, the IPv6 subnets having dhcpv6_stateless
+address mode are thus limited to /64 prefixes.
+
+On DHCPv6_stateful, the IP address is sent on the DHCPV6 Reply message, and the
+network prefixes - and lengths - are communicated through RA messages.
+
+Thus, the guest is able to configure its addresses - and routes - to communicate
+with subnets having longer prefixes.
+
+There is a caveat though - dynamic IP addresses cannot be used, meaning they
+will have to be configured through the networking API [fixed IPs](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail,create-port-detail#create-port)
+create port method - specifying both the subnet id, and the IP address.
+
+#### IPv6 privacy extensions
+
+Privacy extensions are defined in [rfc](https://tools.ietf.org/html/rfc4941).
+They can **only** be applied to stateless address modes, and OVN does not
+implement it.
+
+The only IP address assignment mechanisms currently implemented in OVN use the
+MAC address to configure the host part of the IPv6 address.
```

Comment:
Isn't the IP address assignment on stateless in the guest only? What is the relation to OVN? Is this related to security groups?

Reply:
it is. Thus, I think it will be impossible to configure IP spoofing protection (using IP addresses on the port security when the privacy extensions are used on the guest) when privacy extensions are configured on guests attached to subnets configured in stateless address mode.

I'll add a caveat or table entry to mention this.

Reply:
Done.